classes/controller/AdminController.php

```diff
@@ -2724,6 +2725,8 @@ public function setMedia($isNewTheme = false)
             ]
         );
 
+        Media::addJsDef(['isDebugMode' => _PS_MODE_DEV_]);
```

Comment:
Maybe we could use this opportunity to introduce the `prestashop` variable in BO?
```suggestion
        Media::addJsDef([
            'prestashop' => [
                'debug' => _PS_MODE_DEV_,
            ],
        ]);
```